UnityEngine.AssetBundles/Editor/AssetBundleBrowser/InspectTab/AssetBundleInspectTab.cs

```diff
@@ -223,15 +270,84 @@ public List<string> BundleList
         public void SetBundleItem(InspectTreeItem selected)
         {
             if (selected == null)
+            {
                 m_SingleInspector.SetBundle(null);
+            }
             else
-                m_SingleInspector.SetBundle(selected.bundle, selected.bundlePath);
+            {
+                AssetBundle bundle = this.LoadBundle(selected.bundlePath);
+                m_SingleInspector.SetBundle(bundle, selected.bundlePath);
+            }
         }
 
         [System.Serializable]
         public class InspectTabData
         {
             public string m_BundlePath = string.Empty;
         }
+
+        /// <summary>
+        /// Loads the bundle at the specified path.
+        /// Unloads previously loaded bundles if neccessary.
+        /// </summary>
+        /// <returns>Returns loaded bundle, null if it could not be loaded.</returns>
+        /// <param name="path">Path of bundle to load</param>
+        private AssetBundle LoadBundle(string path)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return null;
+            }
+
+            string bundleName = Path.GetFileNameWithoutExtension(path);
+            AssetBundle bundle = this.GetLoadedBundleByName(bundleName);
+            if (null != bundle)
```

Comment:
Ok, I've got it figured out.
Working well now, will be included in next commit.